_episodes/06-script.md

```diff
@@ -320,6 +320,9 @@ The file `redo-figure-3.sh` now contains:
 ~~~
 {: .source}
 
+
+(goostats is a shell script which calculates some complicated statistics from a datafile -- the first parameter -- and writes them to a file -- the second parameter; the script goodiff compares two datafiles provided as parameters. Nele's superviser provided them without too many explainations.)
```

Comment:
Please change ~parameter~ argument. We've just tidied up the usage of argument vs parameter in the shell lesson #624.
Could you also use backticks for `goostats` and `goodiff`.
And fix ~Nele's~ Nelle's